src/control/server/config.go

```diff
@@ -108,7 +113,7 @@ func (c *Configuration) WithRecreateSuperblocks() *Configuration {
 }
 
 // WithProviderValidator is used for unit testing configurations that are not necessarily valid on the test machine.
-// We use the stub function ValidateNetworkConfigStub to avoid unnecessary failures
+// We use the stub function ValidateProviderStub to avoid unnecessary failures
```

Comment:
Still not really crazy about calling out the test stuff in the comments in production code. IMO this commentary belongs in the tests where the stubs are used.

Reply:
I've removed most of the comment to avoid extra exposure.